### DISCUSSION TITLE
Light theme?

### DISCUSSION THREAD

**Discussion by robrecord** [zellij-org/zellij#2321] [Q&A]
Hi there, I use gruvbox light as my main palette for all apps. I wanted to use this for zellij but it seems the themes are all on dark backgrounds, even when "gruvbox light" is selected. Activating the theme, the surrounding bars at top and bottom don't seem quite right - the dark background under the shortcut panel at the bottom with red text on top is very hard to read.

1. Is there something I can do to tell zellij my theme is light background, or does it auto detect? I saw "theme_hue" in the code but don't know how to use it. It doesn't appear in the theme files.

2. Would that even make any difference, or is it just designed to look this way whatever colours are used?

3. If I can't change the background, can I change that red for something else easier to see against the dark background?

<img width="896" alt="Screenshot 2023-03-23 at 13 52 55" src="https://user-images.githubusercontent.com/38916/227225584-da9eb42e-9391-4328-a4a2-3a976f608eaf.png">

**Comment by LunarEclipse363** [comment]
+1 to this, zellij looks like a nice tool but I can't use it because the UI is borderline unreadable on a light theme terminal.

FYI this is an accessibility issue, light-on-dark text is hard to read for me due to astigmatism ([see this article](https://archive.is/sBeeJ)) so I need to use light themes to be comfortable.

More generally I would also suggest testing color combinations in themes against [WCAG AA contrast guidelines](https://webaim.org/resources/contrastchecker/) to ensure the text is legible for everyone.

**Comment by imsnif** [comment]
Zellij comes with some really nice light themes built in: https://zellij.dev/documentation/theme-list

**Reply by LunarEclipse363** [reply]
I looked through all of these and only `gruvbox-light` seems to work properly and have decent contrast.

I'm wondering if the other themes use some of the shell's colors, because on a light theme terminal some of them will be inverted in what is dark and what is bright.

Here's my alacritty config if you would like to test against it to see what I mean https://gitlab.com/LunarEclipse363/dotfiles/-/blob/fa0a155570bffe6715d4aa366152b1445f0c54e0/user_configs/alacritty.toml

With `everforest-light` for example, it looks like this (note the almost invisible bottom bar labels): 
![image](https://github.com/user-attachments/assets/00de853f-9659-4a31-b5f9-a5afed7c271c)

**Comment by nuvic** [comment]
As a light theme user, I've had to configure a light colorscheme myself. Here's a WIP so far that I prefer to the built in ones.

```kdl
themes {
  flexoki {
    fg "#B7B5AC"
    bg "#FFFCF0"
    black "#FFFCF0"
    red "#AF3029"
    green "#879A39"
    yellow "#D0A215"
    blue "#4385BE"
    magenta "#CE5D97"
    cyan "#3AA99F"
    white "#100F0F"
    orange "#DA702C"
  }
}

theme "flexoki"
```
Add this in your `config.kdl`, and play around with the colors until they match what you want.
